index.html

```diff
@@ -1,9 +1,102 @@
-over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it.
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NPM</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div>
+            <div>
+                <img class="black-heart" src="images/black-heart.png" alt="black heart">
+                <span>Nifty Penguin Magic</span>
+            </div>
+            <nav>
+                <ul>
+                    <li><a href="#">npm Enterprise</a></li>
+                    <li><a href="#">Products</a></li>
+                    <li><a href="#">Solutions</a></li>
+                    <li><a href="#">Resources</a></li>
+                    <li><a href="#">Docs</a></li>
+                    <li><a href="#">Support</a></li>
+                </ul>
+            </nav>
+        </div>
+
+        <div class="search-div">
+           <img class="npm-logo" src="images/npm-logo.png" alt="npm-logo"> 
+           <form class="search-form" action="">
+               <label class="search-label" for="search package">
+                   <img class="search-img" src="images/magnifying-glass.png" alt="magnifying-glass">
+               </label>
+               <input class="search-bar" type="text" placeholder="Search packages">
+               <button class="search-btn" type="submit">Search</button>
+           </form>
+           <a class="join-btn" href="#">Join</a>
+           <a class="log-in-btn" href="#">Log In</a>
+        </div>
+    </header>
+
+    <!-------------------Section 1-------------------------->
+
+    <section id="section-1">
+        <h1>Build amazing things</h1>
+        <article>
+            Essential JavaScript development tools that help you go to market faster and build powerful applications using modern open source code. 
+        </article>
+        <div>
+            <a class="plans-btn" href="#">See plans</a>
+            <a class="free-btn" href="#">Join for free</a>
+        </div>
+    </section>
+
+    <!-------------------Section 2-------------------------->
+
+    <section id="section-2">
+        <img src="images/triangle-hexagon.svg" alt="">
```

Comment:
Remember to always put the description of the image in the "alt" attribute, it is used for blind people to know what is shown in the screen so it's very useful.